modules/koblerBidAdapter.js

```diff
@@ -0,0 +1,226 @@
+import * as utils from '../src/utils.js';
+import {config} from '../src/config.js';
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import {BANNER} from '../src/mediaTypes.js';
+
+const BIDDER_CODE = 'kobler';
+const BIDDER_ENDPOINT = 'https://bid.essrtb.com/bid/prebid_rtb_call';
```

Comment:
Calling this endpoint from a test page returns a CORS error. My test page is running on `http://localhost:3000`. Do I need to change something in order to test this adapter?

Reply:
It should return 204 currently. Can you please try again?

Reply:
I'm now getting the 204. However, can this be updated to return a valid response for placement id `xjer0ch8`?

The test params in your .md file should return data.

Reply:
If there are no bids, we always return 204.

Since Kobler is a contextual platform, we require a page URL that is present in our system in addition to the `placementId`. As a result, it's not possible for any `placementId` we'd give in the .md to return data without also having a matching URL. What do you think we should do in this situation?

Reply:
Is there a way you could set up a test `placementId` that will work with `localhost`?

Reply:
Definitely, but the code that would run would be completely different than what would run for real traffic (matching `placementId` + page URL). Should we still do this?

Reply:
Sorry, I'm not understanding why this would need to be completely different code. It's just in this case the URL would be `http://localhost`.

Reply:
It's because of how mediums and their placements are set up in our system. The requirement for us to bid on a URL is that it's crawled (we know its content), which is obviously not possible for localhost.

The way I tested this adapter is as follows:
- change the `BIDDER_ENDPOINT` to `'https://bid-service.dev.essrtb.com/bid/prebid_rtb_call'` (this is our DEV server which has a medium set up to use Prebid)
- change `refererInfo` to return `'https://www.tv2.no/a/11734615'` as a `referer` (this is the URL of a page belonging to that medium).
- use `ku9i4as30` as a `placementId` and then you should see bids (from a test campaign set up in our DEV environment).

Reply:
I'll try this. Can you add this info to your .md file?

Reply:
I'm able to a response from the dev server. Noticed that it's not gzipped. Are the ones from the prod server gzipped? If not, they should be.

Reply:
I've added the info in the .md file and fixed the missing gzip encoding now. Dev and prod server code is the same, only the data they use is different.